docs/content/guides/author-apps/containers/overview/index.md

```diff
@@ -90,7 +90,7 @@ Refer to the probes section of the [container resource schema]({{< ref "containe
 
 ## Connections
 
```

Comment:
Rereading this now and realizing this is a pretty long run on sentence. Can we split this out into 2+ sentences?